ga4gh/backend.py

```diff
@@ -218,6 +278,14 @@ def addDataset(self, dataset):
         self._datasetNameMap[dataset.getLocalId()] = dataset
         self._datasetIds.append(id_)
 
+    def addOntologies(self, ontologies):
```

Comment:
I think this should be a singlar `addOntology`, this will simplify things and make the Ontology setup the same as Datasets, ReferenceSets etc.